### PR TITLE
Update NoSwap SwapBehavior to LimitedSwap

### PR DESCRIPTION
**Pull request by ehashman**
Per https://github.com/kubernetes/kubernetes/pull/102823/files#r653144961

### CODE REVIEW

File: keps/sig-node/2400-node-swap/README.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ type KubeletConfiguration struct {
 
 type MemorySwapConfiguration struct {
 	// Configure swap memory available to container workloads. May be one of
-	// "", "NoSwap": workloads cannot use swap
+	// "", "LimitedSwap": workload combined memory and swap usage cannot exceed pod memory limit
```
Comment (mrunalp) [inline keps/sig-node/2400-node-swap/README.md:346]: Can we clarify that v1 and v2 behavior will diverge here?

Comment (ehashman) [reply keps/sig-node/2400-node-swap/README.md:346]: It won't diverge in the current implementation, as we're not disallowing swap usage on cgroupsv2. If we want to allow for a "noswap" option in the future, we can add that for v2 only.

Comment (mrunalp) [reply keps/sig-node/2400-node-swap/README.md:346]: Yes, but we can clarify that this LimitedSwap option only applies to cgroups v1 and not v2.

Comment (ehashman) [reply keps/sig-node/2400-node-swap/README.md:346]: It applies to both right now.

Comment (ehashman) [reply keps/sig-node/2400-node-swap/README.md:346]: @mrunalp and I synced offline; I was unaware that cgroupsv2 doesn't support a combined memory+swap limit, and must configure them separately.

This behaviour difference is now addressed below!

**Review by mrunalp — APPROVED** [review]
/lgtm

**Comment by derekwaynecarr** [comment]
Thank you for making the updates.  Thanks @sjenning for sharing your insights with @ehashman and myself during the review.

/approve
/lgtm

**Pull request merged**